.ci/docker/common/install_inductor_benchmark_deps.sh

```diff
@@ -0,0 +1,69 @@
+source "$(dirname "${BASH_SOURCE[0]}")/common_utils.sh"
```

Comment:
Just a note that Docker build process run this script as root, so make sure that you use `as_jenkins` to switch to the Docker user correctly, for example, when running `python install.py`